### PR TITLE
0.9.0 prerelease3

### PR DESCRIPTION
**Pull request by Scarjit**
# 0.9.0 prerelease3

Contains fixes for prerelease 2
Supersedes #974

### CODE REVIEW

File: docs/static/examples/development.yaml [hunk 1]
```diff
@@ -298,7 +298,7 @@ write_files:
       echo "Helm installed!"
 
       echo "Adding helm repo..."
-      if ! helm repo add united-manufacturing-hub https://repo.umh.app/ ; then
+      if ! helm repo add united-manufacturing-hub https://deploy-preview-979--umh-helm-repo.netlify.app/ ; then
```
Comment (JeremyTheocharis) [inline docs/static/examples/development.yaml:301]: see other note

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
let's change the default installation script in development.yaml by removing `--devel`. Then we can also push this prerelease to main.

File: deployment/helm-repo/index.yaml [hunk 2]
```diff
@@ -51,11 +51,11 @@ entries:
     - https://github.com/united-manufacturing-hub/united-manufacturing-hub
     type: application
     urls:
-    - https://repo.umh.app/factorycube-edge-0.7.0.tgz
+    - https://deploy-preview-979--umh-helm-repo.netlify.app/factorycube-edge-0.7.0.tgz
```
[the remark this answers is not in the document]
Comment (Scarjit) [reply deployment/helm-repo/index.yaml]: Resolved in commit below

File: docs/static/examples/development.yaml (hunk 1 above)
Comment (Scarjit) [reply docs/static/examples/development.yaml]: Resolved in commit below

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**